SQL/New_patches/2019-02-27_add_biobank_module_to_schema.sql

```diff
@@ -0,0 +1,505 @@
+-- DROPS --
+
+/*Relational*/
+DROP TABLE IF EXISTS `biobank_container_parent`;
+DROP TABLE IF EXISTS `biobank_container_psc_rel`;
+DROP TABLE IF EXISTS `biobank_specimen_pool_rel`;
+DROP TABLE IF EXISTS `biobank_specimen_type_parent`;
+DROP TABLE IF EXISTS `biobank_specimen_parent`;
+DROP TABLE IF EXISTS `biobank_specimen_type_unit_rel`;
+DROP TABLE IF EXISTS `biobank_specimen_protocol_container_type_rel`;
+DROP TABLE IF EXISTS `biobank_specimen_type_container_type_rel`;
+DROP TABLE IF EXISTS `biobank_specimen_protocol_attribute_rel`;
+
```

Comment:
add those lines:
DROP TABLE IF EXISTS `biobank_specimen_type_attribute_rel`;
DROP TABLE IF EXISTS `biobank_specimen_method_attribute_rel`;

Reply:
These tables don't exit anymore or have changed names. I could include a series of `DROPs` in another file for testing purposes, but I don't think they should go in the patch.